src/components/headers/StyledSectionSubTitle.js

```diff
@@ -5,6 +5,7 @@ import { N2 } from '../../core/style/Colors';
 
 const StyledSectionSubTitle = styled.h2`
   color: ${N2};
+  font-family: 'Open Sans', sans-serif;
```

Comment:
You shouldn't need to do this. Open Sans should be the default font for the entire page. If it's not, this needs to happen inside `index.js` (I think).